xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -1125,7 +1126,9 @@ bool CGUIWindowVideoBase::OnPlayMedia(int iItem, const std::string &player)
   }
   CLog::Log(LOGDEBUG, "%s %s", __FUNCTION__, CURL::GetRedacted(item.GetPath()).c_str());
 
-  if (StringUtils::StartsWith(item.GetPath(), "pvr://recordings/active/"))
+
+  CPVRRecordingsPath path(item.GetPath());
```

Comment:
Note that it does not make much sense to construct a PVR related object every time a video is started. The majority of used does not even use pvr. This is a waste of resources.